menu.ini

```diff
@@ -1069,14 +1146,21 @@ module=am.pl
 action=list_templates
 file=templates=/import.map
 
-[System--Audit]
+[System--Ledger Doctor]
```

Comment:
menu changes which break in Run my Accounts. Also, menu is already there, why that change.